### PR TITLE
fix tasks table

### PR DESCRIPTION
**Pull request by chenxidev1129**
@johnsmith0305 I handle the default value setting in the Dto constructor.

### CODE REVIEW

File: taskmaster-api/Data/Entities/TaskEntity.cs
Comment (johnsmith0305) [inline taskmaster-api/Data/Entities/TaskEntity.cs:1]: @chenxidev1129 Did you check how to handle the default value setting in this Entity as annotation?

Comment (chenxidev1129) [reply taskmaster-api/Data/Entities/TaskEntity.cs:1]: @johnsmith0305 Yes. But there is no way.

Comment (johnsmith0305) [reply taskmaster-api/Data/Entities/TaskEntity.cs:1]: okay

**Review by johnsmith0305 — APPROVED** [review]
We should consider how to manage default value settings in the Entity to ensure that the Migration reflects those changes.

**Pull request merged**